shared/src/business/utilities/pdfs/getPdfJs.test.ts

```diff
@@ -0,0 +1,68 @@
+import { clientSupportsES2022, getPdfJs } from './getPdfJs';
+
+describe('Broweser Compatibility Checks', () => {
+  const mockUserAgent = (userAgent: string) => {
+    Object.defineProperty(navigator, 'userAgent', {
+      configurable: true,
+      value: userAgent,
+    });
+  };
+  describe('getPdfJs - Dynamic Import Failure', () => {
+    jest.mock('pdfjs-dist/legacy/build/pdf.mjs', () => {
+      throw new Error('Dynamic import failed');
+    });
+    it('should throw an error if pdfjs-dist fails to import', async () => {
```

Comment:
While we are here, we might as well expect `clientSupportsES2022` to have been called, too